src/js/constants.js

```diff
@@ -40,5 +40,12 @@ var exports = {
   DNT_POLICY_CHECK_INTERVAL: 1000, // one second
 };
 
+exports.BLOCKED_ACTIONS = {
```

Comment:
Not sure I follow what you mean regarding `const`. Currently nothing that is exported in `constants.js` is of type `const`. Would you explain a bit more about the desired behavior for this?